app/controllers/devise/two_factor_authentication_controller.rb

```diff
@@ -11,10 +11,15 @@ class Devise::TwoFactorAuthenticationController < DeviseController
   def new
     current_user.send_new_otp
     flash[:notice] = t('devise.two_factor_authentication.user.new_otp_sent')
-    redirect_to user_two_factor_authentication_path
+    redirect_to user_two_factor_authentication_path(method: 'sms')
```

Comment:
I think I figured it out. It's to be able to fall back to SMS when you don't have access to your authenticator app, right?. If so, I think a feature test would be helpful here. What do you think about creating a new route for this as opposed to using the `new` action for two different scenarios. A separate route would keep things explicit and separated. Thoughts?